libs/remix-ui/solidity-compiler/src/lib/compiler-container.tsx

```diff
@@ -979,32 +979,47 @@ export const CompilerContainer = (props: CompilerContainerProps) => {
                 ))}
               </select>
             </div>
-            <div className="mt-1 mt-3 border-dark pb-3 ml-4 remixui_compilerConfig custom-control custom-checkbox">
+            <div className="mt-1 mt-3 border-dark pb-3 ml-4 remixui_compilerConfig ">
               <div className="justify-content-between align-items-center d-flex">
-                <input
-                  onChange={(e) => {
-                    handleOptimizeChange(e.target.checked)
-                  }}
-                  disabled={state.useFileConfiguration}
-                  className="custom-control-input"
-                  id="optimize"
-                  type="checkbox"
-                  checked={state.optimize}
-                />
-                <label className="form-check-label custom-control-label" htmlFor="optimize">
-                  <FormattedMessage id="solidity.enableOptimization" />
-                </label>
-                <input
-                  min="1"
-                  className="custom-select ml-2 remixui_runs"
-                  id="runs"
-                  placeholder="200"
-                  value={state.runs}
-                  type="number"
-                  title={intl.formatMessage({ id: 'solidity.inputTitle2' })}
-                  onChange={(e) => onChangeRuns(e.target.value)}
-                  disabled={!state.optimize || state.useFileConfiguration}
-                />
+                <CustomTooltip
+                  placement="top"
+                  tooltipId="configfileOptimisationNumbeTooltip"
+                  tooltipClasses="text-nowrap"
+                  tooltipText={(state.optimize ? "Enable " : "Disable ") + "opcode-based optimizer for the generated bytecode and the Yul optimizer for the Yul code"}
```

Comment:
I think this condition should be reversed as: `state.optimize ? "Disable" : "Enable"`

as if `state.optimize` is true, it means it is already enabled and can be disabled

Reply:
Updated. thanks